wazuh-indexer/config/config.sh

```diff
@@ -0,0 +1,54 @@
+# Wazuh Docker Copyright (C) 2021 Wazuh Inc. (License GPLv2)
+# This has to be exported to make some magic below work.
+export DH_OPTIONS
+
+export NAME=wazuh-indexer
+export TARGET_DIR=${CURDIR}/debian/${NAME}
+
+# Package build options
+export USER=${NAME}
+export GROUP=${NAME}
+export CONFIG_DIR=/etc/${NAME}
+export LOG_DIR=/var/log/${NAME}
+export LIB_DIR=/var/lib/${NAME}
+export PID_DIR=/run/${NAME}
+export INSTALLATION_DIR=/usr/share/${NAME}
+export BASE_DIR=${NAME}-*
+export INDEXER_FILE=wazuh-indexer-base-linux-x64.tar.gz
+export REPO_DIR=/unattended_installer
+
+
+rm -rf ${INSTALLATION_DIR}/
+
+curl -o ${INDEXER_FILE} https://s3.amazonaws.com/warehouse.wazuh.com/indexer/${INDEXER_FILE}
```

Comment:
Nice!, change the path to pre-release `https://packages-dev.wazuh.com/stack/indexer/base/wazuh-indexer-base-4.3.0-linux-x64.tar.xz` note that now we are using XZ